app/src/main/java/ar/nablab/mimerendero/views/MainActivity.kt

```diff
@@ -0,0 +1,83 @@
+package ar.nablab.mimerendero.views
+
+import android.os.Bundle
+import android.util.Log
+import androidx.activity.viewModels
+import androidx.annotation.VisibleForTesting
+import androidx.appcompat.app.AppCompatActivity
+import ar.nablab.mimerendero.R
+import ar.nablab.mimerendero.dtos.Place
+import ar.nablab.mimerendero.viewModels.MapViewModel
+import com.google.android.gms.location.FusedLocationProviderClient
+import com.google.android.gms.location.LocationServices
+import com.google.android.gms.maps.CameraUpdateFactory
+import com.google.android.gms.maps.GoogleMap
+import com.google.android.gms.maps.OnMapReadyCallback
+import com.google.android.gms.maps.SupportMapFragment
+import com.google.android.gms.maps.model.LatLng
+import com.google.android.gms.maps.model.MarkerOptions
+import dagger.hilt.android.AndroidEntryPoint
+
+@AndroidEntryPoint
```

Comment:
I was expecting the main dagger component object initialization here. Does AndroidEntryPoint accomplishes the same?

Reply:
Yes, this is Hilt, not dagger, so this is the way to say which is the main entry of the app.
[https://developer.android.com/training/dependency-injection/hilt-android](https://developer.android.com/training/dependency-injection/hilt-android)